soweego/importer/base_dump_extractor.py

```diff
@@ -32,3 +32,18 @@ def get_dump_download_urls(self) -> Iterable[str]:
         :rtype: str
         """
         raise NotImplementedError
+
+    def _commit_entity(self, db_manager, entity):
+        success = True
+        session = db_manager.new_session()
+        try:
+            session.add(entity)
+            session.commit()
+        except:
+            session.rollback()
+            success = False
+        finally:
+            session.close()
+
+        if not success:
+            self._commit_entity(db_manager, entity)
```

Comment:
Quite brute force: is this actually working?

Reply:
No, it didn't. I'm going to add some delay before retrying.